shared/settings/feedback/index.tsx

```diff
@@ -37,6 +38,10 @@ class Feedback extends React.Component<Props, State> {
     sendLogs: true,
     showSuccessBanner: false,
   }
+  componentWillMount(): void {
```

Comment:
This is legacy now: https://reactjs.org/docs/react-component.html#legacy-lifecycle-methods

`componentDidMount()` would work I think

Reply:
You're not supposed to `setState` in `componentDidMount` because that triggers a double render. Looks like I need a `constructor()`.